lang/en.yml

```
@@ -0,0 +1,101 @@
+MAIN_MENU: "Main menu"
+WELCOME: "Welcome to {0}"
+APP_NAME: "Easy Legendary"
+UNCOMPILED_VERSION_DETECTED: "Uncompiled version of {0} detected."
+UNSTABLE_BUILD_WARNING: "This build may be unstable."
+SELECT_VALID_OPTION: "Select a valid option!"
+EMPTY_PLACEHOLDER: "Nothing here!"
+SELECT_ACTION: "Select what do you want to do:"
+
+ACTIONS:
+    START: "start"
+    IMPORT: "import"
+    IMPORT_GAME_IN_PATH: 'import the game "{0}", located in "{1}"'
+    INSTALL: "install"
+    INSTALL_GAME_IN_PATH: 'install "{0}" in "{1}"'
+    MOVE_GAME_TO_PATH: 'move the game "{0}" to "{1}"'
+    UNINSTALL: "uninstall"
+    UNINSTALL_GAME: "uninstall {0}"
+    UPDATE: "update"
+    MOVE: "move"
+    VERIFY: "verify the files of"
+    VERIFY_GAME: 'verify the files of "{0}"'
+    FIND: "find the files of"
+    SHORTCUT: "make a shortcut for"
+    INFO: "see info for"
+    UPDATE_TO_VERSION: "update to {0}"
+    SYNC_GAMES: "sync games with the Epic Games Launcher"
+    SYNC_ONLINE_SAVES: "sync your online saves"
+    DELETE_CORRUPTED_ONLINE_SAVES: "delete corrupted online saves"
+
+LIST_OWNED_GAMES: "List owned games"
+LIST_INSTALLED_GAMES: "List installed/imported games"
+START_GAME: "Start a game"
+INSTALL_GAME: "Install a game"
+UPDATE_GAME: "Update a game"
+IMPORT_GAME: "Import a installed game"
+UNINSTALL_GAME: "Uninstall a game"
+BROWSE_GAME_FILES: "Browse within game's files"
+SHOW_GAME_INFO: "Show info about a game"
+SYNC_CLOUD_SAVES: "Sync cloud saves"
+SYNC_WITH_EGL: "Sync games with Epic Games Launcher"
+VERIFY_GAME_FILES: "Verify a game's files"
+MOVE_GAME: "Move a game to another folder"
+CREATE_SHORTCUT: "Create a shortcut"
+REFRESH_CACHE: "Refresh/Update the cache"
+CHECK_FOR_UPDATES: "Check for updates"
+EXIT: "Exit"
+
+PRESS_ENTER_TO_RETURN_TO_MAIN_MENU: "Press enter to return to main menu..."
+SELECT_THIS_ITEM_TO_EXIT: "Select this item to exit..."
+SAVING_CACHE: "Saving cache..."
+GOODBYE: "Goodbye!"
+TYPE_A_VALID_PATH: "Type a valid path"
+GAME_NOT_FOUND: "Game not found!"
+
+FILE_EXPLORER_WINDOW: "A file explorer window should have opened with the game's files."
+UP_TO_DATE: "There isn't any update available! You are up to date!"
+UPDATE_AVAILABLE: "There is an update available!"
+CHANGELOG: "Changelog"
+USING_UNCOMPILED_VERSION: "You are using a uncompiled version of easy-legendary."

```

```suggestion
USING_UNCOMPILED_VERSION: "You are using a uncompiled version of {0}."
```
Should be replaced with {0} and then replace {0} in the code with the localized APP_NAME text.